src/action-creators/fontSize.ts

```diff
@@ -1,11 +1,14 @@
 import { Thunk } from '../types'
+import { storage } from '../util/localStorage'
 
 /** Sets fontSize in state and localStorage.. */
-const fontSize = (value: number): Thunk => (dispatch, getState) => {
-  dispatch({ type: 'fontSize', value })
-  setTimeout(() => {
-    localStorage.fontSize = value
-  })
-}
+const fontSize =
+  (value: number): Thunk =>
+  (dispatch, getState) => {
+    dispatch({ type: 'fontSize', value })
+    setTimeout(() => {
+      storage.fontSize = value
```

Comment:
This doesn't work on web. It saves to `storage.fontSize`, not the browser's `localStorage`. I think you need `storage.setItem('fontSize', value)`.

You can reproduce this by increasing the font size using the links in the footer and then refreshing the page.